addon/components/admin-input.js

```diff
@@ -1,18 +1,21 @@
 import Ember from 'ember';
 
-var set = Ember.set;
-var get = Ember.get;
-var computed = Ember.computed;
+const set = Ember.set;
+const get = Ember.get;
+const computed = Ember.computed;
```

Comment:
Is not working with 
```javascript
const {
  computed
} = Ember;
```
This make a error "Unexpected token {".
I did directly Ember.computed as is other files, and it works.
What I do? Should I make a new PR for this?